src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -55,6 +55,11 @@ internal sealed partial class ConsoleHost
         internal const int ExitCodeCtrlBreak = 128 + 21; // SIGBREAK
         internal const int ExitCodeInitFailure = 70; // Internal Software Error
         internal const int ExitCodeBadCommandLineParameter = 64; // Command Line Usage Error
+        private const uint SPI_GETSCREENREADER = 0x0046;
+
+        [DllImport("user32.dll", SetLastError = true)]
```

Comment:
We could remove `SetLastError = true` - we don't use it.

Reply:
I prefer to keep it, even it's not used in this case. People copy code, and this is a good practice for PInvoke.